scripts/jurisdiction_data.sql

```diff
@@ -0,0 +1 @@
+insert into jurisdiction(jurisdiction_id, jurisdiction_name, url) values (1, 'SSCS', 'http://SSCS.endpoint');
```

Comment:
This is fine, and in accordance with HLD v0.3. I suspect we'll need to add a "description" field. The external jurisdiction will not know our internal ID for this entity so should use a short code, in this case the jurisdiction_name. Then we'll have a description field to tell us what this entity is.

Fine you to do this in another branch.